README.md

```diff
@@ -20,6 +20,60 @@ This repository contains Helm charts served by Rancher Apps & Marketplace.
 
 ---
 
+### New Out-Of-Band Release Process
+
+This is only valid for `prod-v2.*` branches.
+Since this implementation, all teams may release each chart when they want.
+
+##### Overview of the process:
+1. Assuming you have a chart ready and merged on `dev-v2.*` branch.
+2. On your local machine: `fetch`, `pull` and `checkout` to `prod-v2.*`
+3. Create a new branch from `prod-v2.*`
+4. Execute `make forward-port`
+5. Clear your `release.yaml` file, leave only your chart that will be released
```

Comment:
this should happen before `make forward-port` because `make forward-port` adds the chart to `release.yaml`